packages/dataviews/src/dataviews-layouts/utils/get-clickable-item-props.ts

```diff
@@ -25,7 +25,11 @@ export default function getClickableItemProps< Item >( {
 			onClickItem( item );
 		},
 		onKeyDown: ( event: React.KeyboardEvent ) => {
-			if ( event.key === 'Enter' || event.key === '' ) {
+			if (
+				event.key === 'Enter' ||
+				event.key === '' ||
```

Comment:
What's the key related to empty space?

Reply:
I wonder if the empty string was just mistyping for SPACE key? cc @youknowriad @gigitux who were involved in this, if my memory doesn't fail me.